include/deal.II/matrix_free/operators.h

```diff
@@ -24,6 +24,7 @@
 
 #include <deal.II/lac/diagonal_matrix.h>
 #include <deal.II/lac/la_parallel_vector.h>
+#include <deal.II/lac/vector_view.h>
```

Comment:
This should fix the two arpack tests here:
http://cdash.kyomu.43-1.org/viewTest.php?onlydelta&buildid=2317